### PR TITLE
feat(webserver): Send email on user signup

### PR DESCRIPTION
**Pull request by boxbeam**
Fixes TAB-535

### CODE REVIEW

File: ee/tabby-email/emails/signup_success.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+} from "@react-email/components";
+import * as React from "react";
+import RootLayout from "../components/root-layout";
+
+interface PasswordReset {
+  email?: string;
+  resetLink?: string;
+}
+
+export const PasswordReset = ({
```
Comment (wsxiaoys) [inline ee/tabby-email/emails/signup_success.tsx:14]: please update variable name accordingly?

File: ee/tabby-email/emails/signup_success.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  Hr, Link, Section,
+  Text
+} from "@react-email/components";
+import * as React from "react";
+import RootLayout from "../components/root-layout";
+
+interface PasswordReset {
+  email?: string;
+  resetLink?: string;
+}
```
Comment (wsxiaoys) [inline ee/tabby-email/emails/signup_success.tsx:12]: ditto, please also update field

File: ee/tabby-email/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## Creating email templates for Tabby
```
Comment (wsxiaoys) [inline ee/tabby-email/README.md:1]:
```suggestion
## Add new email template for Tabby

```

**Review by wsxiaoys — COMMENTED** [review]
Please also provide screenshot for email received.

**Comment by boxbeam** [comment]
image

File: ee/tabby-email/emails/signup_success.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+          {link}
+        </Link>
+      </Text>
+    </RootLayout>
+  );
+};
+
+SignupSuccess.PreviewProps = {
+  email: "user@tabbyml.com",
+  link: "http://localhost:8080/auth/reset-password?code={{CODE}}",
```
Comment (wsxiaoys) [inline ee/tabby-email/emails/signup_success.tsx:40]: also update link here?

File: ee/tabby-email/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+## Add new email template for Tabby
+- Navigate to `ee/tabby-email/emails`
```
Comment (wsxiaoys) [inline ee/tabby-email/README.md:2]:
```suggestion
## Add new email template for Tabby

- Navigate to `ee/tabby-email/emails`
```

**Review by wsxiaoys — COMMENTED** [review]
Otherwise LGTM

**Pull request merged**